README.md

```diff
@@ -44,4 +46,5 @@ Give a ⭐️ if you like this project!
 Hat tip to Creative Commons license of the design and Cindy Shin for inspiration.
```

Comment:
[OPTIONAL] I am only marking this as optional because the previous reviewer missed it. It is a requirement for you to add a link to Cindy's design o Behance in your acknowledgements section and not just her name.

Reply:
I am literally going to repeat this module, I am hurt

Reply:
@fobadara I can't believe I came all the way this far to fail the Capstone